docs/connectors/index.mdx

```diff
@@ -161,6 +167,85 @@ specified.
 See [Delta Lake output connector documentation](/connectors/sinks/delta)
 for an example of configuring the output buffer.
 
+## Orchestration (starting and pausing)
+
+:::note
+
+Only input connectors currently support orchestration.
+
+:::
+
```

Comment:
```suggestion
Connector Orchestration enables users to activate or deactivate input connectors on demand, giving them control over the timing and order of data ingestion from multiple sources. It can, for example, be used to backfill a pipeline with historical data from a database or data lake before switching over to real-time ingestion from a streaming source like Kafka.
```